components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/internal/OAuth2ServiceComponentHolder.java

```diff
@@ -141,6 +142,16 @@ public static void setConsentedTokenColumnEnabled(boolean consentedTokenColumnEn
         OAuth2ServiceComponentHolder.consentedTokenColumnEnabled = consentedTokenColumnEnabled;
     }
 
+    public static boolean isAcrColumnEnabled() {
```

Comment:
Lets add javadoc comments for public methods

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1984/commits/59ba3e10b73d02dcf0d68de230df5add71a2ab97